File: src/main/java/bilibili/lingaly/tutorial/ModRegister.java
package bilibili.lingaly.tutorial;

import bilibili.lingaly.tutorial.item.ModItems;
import net.fabricmc.fabric.api.registry.FuelRegistry;

public class ModRegister {

    public static void modRegister(){
        registerFuel();
    }

    private static void registerFuel() {
        FuelRegistry fuelRegistry = FuelRegistry.INSTANCE;

        //1秒=20tick，200tick可烧纸一个物品(即10秒)
        fuelRegistry.add(ModItems.PROBE, 100);
    }

}

File: src/main/kotlin/bilibili/lingaly/ktmccouse/ModRegister.kt
package bilibili.lingaly.ktmccouse

import bilibili.lingaly.easyfabricregisterlib.EFRLModRegister
import bilibili.lingaly.ktmccouse.items.ModItems

object ModRegister {

    fun modRegister(){
        registerFuel()
    }

    private fun registerFuel() {
        EFRLModRegister.registerFuel(
            ModItems
                .PROBE,
            200
        )
    }

}